### ISSUE TITLE
Update repo to v0.14.0

### ISSUE THREAD

**Issue by JordanMartinez** [hdgarrood/purescript-benchotron#23]
Once this issue gets resolved, we can re-enable the benchmarks in `purescript-free` again. See purescript/purescript-free#107

Note: this issue likely won't be done until after core repos are updated.

**Comment by purefunctor** [comment]
A bit of a bump on this, e64664d successfully builds on the current package set; I'm also currently using it for [ssrs](https://github.com/PureFunctor/purescript-ssrs). I think the CI failure is from the outdated pulp, other than that, is there anything else needed for release? I'm not entirely sure how `pulp`+`bower` works, so would the versions in `bower.json` need to be updated as well?

**Comment by hdgarrood** [comment]
Yes, the versions in `bower.json` need to be updated. I will make a release if you can make a PR which gets CI passing again.

**Comment by hdgarrood** [comment]
I don't have time to cut a release right now, please ping me if I haven't done it by next week.

**Comment by flip111** [comment]
@hdgarrood ping. The package is still on release 7.0.1 from Jun 8, 2019

**Comment by hdgarrood** [comment]
Ah, oops. This repo is unmaintained right now - I'm probably not going to push or cut any more updates. Feel free to fork it if you're still using it

**Comment by flip111** [comment]
@hdgarrood can you transfer ownership
and/or put a line in the readme that it is unmaintained
and/or put the repo on read-only github setting?

it's not so clear how to do package takeover on pursuit and it's best to ask repo owner

**Comment by hank-der-hafenarbeiter** [comment]
@hdgarrood Any news? have you transfered ownership yet?

**Comment by hdgarrood** [comment]
No, I never said I would transfer ownership and I do not intend to do that.
